grpcclient/grpcclient.go

```diff
@@ -27,6 +28,9 @@ type Config struct {
 	BackoffOnRatelimits bool           `yaml:"backoff_on_ratelimits" category:"advanced"`
 	BackoffConfig       backoff.Config `yaml:"backoff_config"`
 
+	InitialStreamWindowSize     flagext.Bytes `yaml:"initial_stream_window_size" category:"advanced"`
+	InitialConnectionWindowSize flagext.Bytes `yaml:"initial_connection_window_size" category:"advanced"`
```

Comment:
These are very advanced settings, which I'm not 💯 sure we should even expose to the final user. I'm OK having us experimenting with it, but I'm wondering if it's better to categorise them as "experimental" so that we can eventually remove them anytime with no notice.